File: backend/src/main/java/com/project2/domain/member/dto/FollowRequestDto.java
package com.project2.domain.member.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import com.project2.domain.member.entity.Follows;

@Getter
@Setter
@NoArgsConstructor
public class FollowRequestDto {
    private Long followerId;
    private Long followingId;

//    public FollowRequestDto(Follows follows) {
//
//        this.followerId = follows.getFollower().getId();
//        this.followingId = follows.getFollowing().getId();
//    }
}

File: backend/src/main/java/com/project2/domain/member/dto/FollowRequestDto.kt
package com.project2.domain.member.dto

import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@NoArgsConstructor
class FollowRequestDto {
    var followerId: Long? = null
        set(value) {
            field = value
        }
    var followingId: Long? = null
        set(value) {
            field = value
        }

//    public FollowRequestDto(Follows follows) {
    //
    //        this.followerId = follows.getFollower().getId();
    //        this.followingId = follows.getFollowing().getId();
    //    }
}
